spk/tt-rss/src/web/tt-rss.conf

```diff
@@ -0,0 +1,16 @@
+Alias "/tt-rss" "/var/services/web/tt-rss"
+
+<Proxy "unix:/run/php-fpm/php-com-synocommunity-packages-tt-rss.sock|fcgi://tt-rss">
+    ProxySet timeout=3600
+</Proxy>
+
+<Directory "/var/services/web/tt-rss">
+
+    <FilesMatch "\.(php[345]?|phtml)$">
```

Comment:
I'm not sure there is a need for supporting php3, php4 and php5 files... From what I recall I've only seen php files

Reply:
This is for the Apache config. The format was taken from configs generated by DSM so I'd prefer to leave them as-is